entity/IntegrationEntities.xml

```diff
@@ -7,13 +7,7 @@
         <field name="initDate" type="date-time"/>
     </entity>
     <extend-entity entity-name="SystemMessage" package="moqui.service.message">
-        <field name="sentProductId" type="text-medium"/>
-        <field name="sentProductCategoryId" type="text-medium"/>
-        <field name="sentPartyId" type="text-medium"/>
-        <field name="sentContactMechId" type="text-medium"/>
-        <field name="shopifyOrderId" type="text-medium"/>
-        <field name="shopifyDisplayOrderId" type="text-medium"/>
-        <field name="sentShipmentId" type="text-medium"/>
+        <field name="productId" type="text-medium"/>
     </extend-entity>
     <extend-entity entity-name="Shipment" package="mantle.shipment">
         <field name="shopifyShipmentStatusId" type="text-short"/>

```

Comment:
Could a 1-1 mapping of shopify shipment status to the mantle shipment statuses? If so that might be a good way to go, assuming there isn't a good reason for this.